core/templates/pages/exploration-editor-page/editor-tab/graph-directives/exploration-graph.directive.ts

```diff
@@ -53,8 +53,7 @@ angular.module('oppia').directive('explorationGraph', [
           // We hide the graph at the outset in order not to confuse new
           // exploration creators.
           ctrl.isGraphShown = function() {
-            return Boolean(ExplorationStatesService.isInitialized() &&
-              ExplorationStatesService.getStateNames().length > 1);
+            return Boolean(ExplorationStatesService.isInitialized());
```

Comment:
Have you checked if this does not cause other things to break? ie. Check why the additional check was added in the first place and verify that your fix does not break that behaviour.

Reply:
I couldn't find a reason for it apart from aesthetics, since a 'graph' is relevant only after 1 state, but in this case, errors are shown in the graph even if there is only one state, so I enabled it there also.
I tried the editor with this and there seemed to be no issues.

Reply:
ok sg!